hail/python/test/hail/helpers.py

```diff
@@ -127,6 +128,19 @@ def wrapper(func, *args, **kwargs):
 
     return wrapper
 
+def fails_local_backend():
+    from hail.backend.local_backend import LocalBackend
+
+    @decorator
+    def wrapper(func, *args, **kwargs):
+        if isinstance(hl.utils.java.Env.backend(), LocalBackend):
+            with pytest.raises(BaseException):
+                return func(*args, **kwargs)
+        else:
+            return func(*args, **kwargs)
+
+    return wrapper
```

Comment:
It looks like there's an official pytest decorator to mark a test as expected to fail, which the test reports treat specially. https://docs.pytest.org/en/latest/skipping.html#xfail-mark-test-functions-as-expected-to-fail

Using that, I think you could write this
```
fails_local_backend = pytest.mark.fail(isinstance(hl.utils.java.Env.backend(), LocalBackend), reason="doesn't yet work on local backend", strict=True)
```

Reply:
Thoughts on this? It seems nice to have the pytest report list separate counts for “expected to fail” and “unexpectedly passing”.

Reply:
oh, nice! I couldn't find this when I was poking around for something better.

Reply:
using xfail now, way better!